main/index.js

```diff
@@ -14,6 +14,7 @@ const { setupAppMenu } = require('./app-menu')
 
 const { ActivityLog } = require('./activity-log')
 const { ipcMain } = require('electron/main')
+const { JobCounter } = require('./job-counter')
```

Comment:
I did not spend too much time picking the name "job counter". I am happy to use a more fitting name if we can find one.